api/v1alpha1/fluxdeployer_types.go

```diff
@@ -14,9 +15,21 @@ type FluxDeployerSpec struct {
 	// +required
 	SourceRef ObjectReference `json:"sourceRef"`
 
+	// The interval at which to reconcile the Kustomization.
+	// +kubebuilder:validation:Type=string
+	// +kubebuilder:validation:Pattern="^([0-9]+(\\.[0-9]+)?(ms|s|m|h))+$"
+	// +required
+	Interval metav1.Duration `json:"interval"`
```

Comment:
Technically, this is a breaking change as this moved here. The interval would just be ignored that was set in the customization template before.

Reply:
I think you should mark it as deprecated then.